install_files/ansible-base/roles/validate/files/validate-supported-locales.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# Verifies a all arguments match a translation directory
+#
+# Exit codes
+# 0 - all arguments match a translation directory
+# 1 - otherwise
+
+for translation in "$@" ; do
```

Comment:
I think we want this to be `for translation in $@ ; do` - as it is right now the additional quotes mean that bash doesn't split on the whitespace in the string, e.g.: 

```
amnesia@amnesia:~/Persistent/securedrop$ ./install_files/ansible-base/roles/validate/files/validate-supported-locales.sh "foobar ar"
securedrop/translations/foobar ar does not exist
amnesia@amnesia:~/Persistent/securedrop$ ./install_files/ansible-base/roles/validate/files/validate-supported-locales.sh foobar ar
securedrop/translations/foobar does not exist
```

(this is the cause of the failure I saw for the test case `ar fr_FR`)

Reply:
Fixed. I wonder how it worked in the past. I'm tempted to blame ansible changing quoting rules. But not motivated to actually look for a proof because it's such a simple fix ;-)